graylog2-web-interface/src/views/components/aggregationbuilder/AggregationBuilder.jsx

```diff
@@ -31,7 +31,7 @@ export type VisualizationComponentProps = {
   config: AggregationWidgetConfig,
   data: { [string]: Rows, events?: Events },
   editing?: boolean,
-  effectiveTimerange: TimeRange,
+  effectiveTimerange: AbsoluteTimeRange,
```

Comment:
As far as I know the effective time range is always an `AbsoluteTimeRange`. Or is there a case were it can be another type?